src/shogun/base/class_list.cpp.py

```diff
@@ -62,7 +62,8 @@
         "NullFileSystem", "FilterVisitor", "RandomMixin", "MaxCrossValidation",
         "StreamingDataFetcher", "MaxMeasure", "MaxTestPower",
         "MedianHeuristic", "WeightedMaxMeasure", "WeightedMaxTestPower",
-        "Seedable", "ShogunEnv", "ShapeVisitor", "FeatureImportanceTree"]
+        "Seedable", "ShogunEnv", "ShapeVisitor", "FeatureImportanceTree",
```

Comment:
what were these for again?

Reply:
Need to remove `GaussianWidthAutoInit` from class_list.cpp (we did the same with the gamma "GammaFeatureNumberInit"), as it is not an SGObject